build/build.php

```diff
@@ -65,7 +65,7 @@ function usage($command)
 // Parse input options
 $options = getopt('', ['help', 'remote::', 'exclude-zip', 'exclude-gzip', 'exclude-bzip2']);
 
-$remote       = isset($options['remote']) ? $options['remote'] : false;
+$remote       = $options['remote'] ?? false;
```

Comment:
@PhilETaylor The null coalesce operator is not supported e.g. on PHP 5.3, so we can't do this on staging.

Reply:
Yes please revert it here that script is intended to run on php 5.4 right now. I personally have nothing against changing that but that would be out of scope of this PR.

Reply:
ah sorry - aint no one got time for staging anymore ;-) I'll revert that bit

Reply:
Pity we have to support these stone aged PHP versions ... but that's another issue 😄 Personally I live the Elvis and null coalesce operators.